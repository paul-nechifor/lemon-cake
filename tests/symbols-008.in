(list (quote aa) (quote bb) (quote cc))
  (list (quote a) (quote c))
( list (quote a) (quote b) (quote c) ) # Nothing here
(list (quote cc) (quote dd))
