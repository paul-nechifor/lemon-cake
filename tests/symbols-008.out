(aa bb cc)
(a c)
(a b c)
(cc dd)
